lib/aur-sync

```diff
@@ -58,15 +58,24 @@ db_fill_empty() {
     }'
 }
 
-# fields: $1 pkgname, $2 pkgbase, $3 pkgver
+# files: $1 pkgname\tpkgbase $2 pkgname (order by $2)
 select_pkgbase() {
     awk 'NR == FNR {
-            a[$0]; next
-        } {
-            if($1 in a && !($2 in seen)) {
-                print $0; seen[$2]
-            }
-        }' "$@"
+	map[$1] = $2
+	next
+    } {
+	if ($1 in map) {
+	    base = map[$1]
```

Comment:
Ew, mixed indentation :stuck_out_tongue:

Reply:
Man, emacs ignores `dir-locals` when switching to `awk` mode for some reason :disappointed: